app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -160,10 +160,8 @@ class MenuDrawerFragment : Fragment() {
         mainViewModel.listenToMailboxes().observe(viewLifecycleOwner) { mailboxes ->
             val sortedMailboxes = mailboxes.filterNot { it.mailboxId == AccountUtils.currentMailboxId }
             addressAdapter.setMailboxes(sortedMailboxes)
-            if (sortedMailboxes.isEmpty()) {
-                addressesList.isGone = true
-                addressesListDivider.isGone = true
-            }
+            addressesList.isGone = sortedMailboxes.isEmpty()
+            addressesListDivider.isGone = sortedMailboxes.isEmpty()
```

Comment:
use `with` here for `sortedMailboxes.isEmpty()`

Reply:
I'm really not fond of those `this`.

<img width="399" alt="image" src="https://user-images.githubusercontent.com/1788629/195775171-29b80cda-9258-4c83-9267-b50a523dbd01.png">

Reply:
And so you prefer to calculate twice in a row that the list is empty?
Otherwise if it's the `this` that bothers you then use a variable